modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you must redirect network traffic of your stateless applications from the source cluster to the target cluster.
+
+The methods for redirecting network traffic are based on the following assumptions:
+
+* The application pods are running on both the the source and target clusters.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If you require secure connections through the source cluster's DNS domain, the target router default certificate includes the source cluster wildcard `Subject Alternative Name`.
```

Comment:
If you can find other instances of Subject Alternative Name in backticks, I guess leave as you have.  I didn't find any and I don't see why it would be styled as inline code--it's not a command, system item, or one of the other items in the [markup quick reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Reply:
@mikemckiernan @sbeskin-redhat 

"Subject Alternative Name" is the name of the SAN extension. The certificate field is `subjectAltName`. See https://support.dnsimple.com/articles/what-is-ssl-san/ 

For the sake of consistency, "source cluster's DNS domain" > "source cluster DNS domain" like "source cluster hostname".

That sentence has a lot of nouns and adjectives strung together at the end. They should be broken up for clarity because the meaning is not at all clear. Suggested change:

"If you require secure connections through the source cluster DNS domain, the default CA certificate of the target router must include a Subject Alternative Name for the DNS wildcard record of the source cluster."

An SSL CA certificate can have multiple SANs. The user needs to make sure that the one of those SANs is for the DNS wildcard record, so that the target router recognizes secure requests from the source cluster.

Reply:
I just noticed that the SAN in the target router certificate is mentioned immediately above ("For secure HTTPS access, perform the following additional step:"), so this is repetition. I think that you can make this line a lot shorter, since you just provided a procedure.

"For HTTPS, the target router CA certificate contains a Subject Alternative Name for the wildcard DNS record of the source cluster."